plugins/GSdx/Renderers/DX11/GSDevice11.h

```diff
@@ -211,25 +217,29 @@ class GSDevice11 final : public GSDevice
 				uint32 fbmask:1;
 
 				// Blend and Colclip
+				uint32 hdr:1;
 				uint32 blend_a:2;
-				uint32 blend_b:2;
-				uint32 blend_c:2;
+				uint32 blend_b:2; // bit30/31
+				uint32 blend_c:2; // bit0
 				uint32 blend_d:2;
 				uint32 clr1:1;
-				uint32 hdr:1;
+
```

Comment:
hdr was fine in original spot.

Reply:
I changed it for better alignment. https://pastebin.com/5dDQXXfn
I could pad it instead.